### ISSUE TITLE
baseline results

### ISSUE THREAD

**Issue by marvis** [huanghoujing/AlignedReID-Re-Production-Pytorch#5]
Hi @huanghoujing,

When training on market1501 using resnet50, the results in open-reid is 85.1 for triplet and 81.4 for softmax. But your baseline result (GL-NF) is 81.53%. Do you use softmax training as the baseline?

See https://cysu.github.io/open-reid/examples/benchmarks.html

Great work!
Best,

**Comment by huanghoujing** [comment]
GL-NF only uses Triplet Loss with Global distance. The difference to open-reid triplet loss example includes 
- whether to use standard Market1501 train/test split, false in open-reid, but true here
- whether to use an extra embedding layer (128-dim) after average pooling (2048-dim), used in open-reid, but here not
- whether to normalize the feature before computing triplet loss and for testing, done in GL-NF. This has little influence when triplet loss is only trained with global distance.
- the epoch at which the learning rate starts to decay, epoch 100 for open-reid and 75 here

Thanks :)

**Comment by marvis** [comment]
Thanks for sharing your training details!  

However I have different experiment results comparing with your fifth item. After feature normalization before computing triplet loss, the CMC score decreases ~8%.  And I find that with or without extra embedding layer doesn't affect the accuracy.  So I doubt whether normalization caused the different baseline result?

Best,

**Comment by huanghoujing** [comment]
1. I found that for `Global Distance + Triplet Loss`, i.e. the commonly used triplet loss paradigm, normalizing feature or not has little influence. 
2. Without normalization, I found it ok to train `Global Distance + Local Distance + Triplet Loss`. So later experiments only consider not normalizing features.
3. By `decreases ~8%`, do you mean `GL-LL-NF-LHSFLD-TWLD` is worse than `GL-LL-NNF-LHSFLD-TWLD` by `~8` CMC Rank-1 points?
4. BTW, the percentage of *satisfying margin*, i.e. `distance(anchor, positive) + margin < distance(anchor, negative)` is a good signal to check the status of convergence. It should be at least over `90%` for both global and local distance. Do you find something abnormal in this signal at the end of training?

**Comment by huanghoujing** [comment]
With more experiments, I find that for vanilla triplet loss, with and without normalizing feature indeed makes difference, the later being better. Some results are in the provided [Excel file](AlignedReID-Scores.xlsx).

Thank you for discussion.
